content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,245 @@
+---
+rank: 4
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+alias_paths:
+  - /docs/box-content-explorer
+  - /docs/content-explorer
+---
+
+# Content Explorer - metadata view
+
+With Content Explorer you can also display files and folders
+based on their metadata.
+The metadata view uses [metadata template][template] and [metadata query][metadata-query] to
+find the data you want to display.
+
+## Prerequisites
+
+* Read the [Content Explorer][explorer] guide.
+* Check the [metadata terminology][terminology].
+* Check the information on [metadata queries][metadata-query].
+
+## Create and configure an app
+
+1. [Create a Box app][box-app].
+2. Add the local development address in the CORS Domains: ![CORS Domains](./images/box-app-cors.png)
+3. Generate a [developer token][token].
+
+## Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
+2. Apply an already created template to a Box folder. Make sure you enable the
+cascade policy. For detailed instructions, see
+[instructions on customizing and applying templates][apply-templates].
+
+### Display name and key parameters
+
+* The `displayName` parameter is the display name of the template visible
```

Comment:
it's actually the metadata key, not the template

Reply:
actually, it's `[fields].displayName`, do we need the display name in this case?